Makefile

```diff
@@ -1,95 +1,105 @@
-.PHONY: minikube-start minikube-stop minikube-delete minikube-dashboard minikube-tunnel init-local init-staging init-production
-
+.PHONY: help
+help: # @HELP Print this message
+help:
+	@echo "TARGETS:"
+	@grep -E '^.*: *# *@HELP' $(MAKEFILE_LIST)    \
+	    | awk '                                   \
+	        BEGIN {FS = ": *# *@HELP"};           \
+	        { printf "  %-20s %s\n", $$1, $$2 };  \
+	    '
+
+.PHONY: minikube-start
```

Comment:
We probably could also do a `minikube-%` target, but then people could start calling through to a lot of things we might not want them to call through to. Not sure.